src/desktop/apps/conversations/server.tsx

```diff
@@ -0,0 +1,55 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { stitch } from "@artsy/stitch"
+import { conversationRoutes } from "reaction/Apps/Conversation/routes"
+import React from "react"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import express, { Request, Response, NextFunction } from "express"
+
+export const app = express()
+
+app.get(
+  "/user/conversations",
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const context = buildServerAppContext(req, res, {})
+      const {
+        bodyHTML,
+        redirect,
+        status,
+        headTags,
+        styleTags,
+        scripts,
+      } = await buildServerApp({
```

Comment:
Will want to add `buildServerAppContext`: https://github.com/artsy/force/blob/master/src/desktop/apps/experimental-app-shell/server.tsx#L56

Reply:
interesting in my copy this was happening few lines up https://github.com/artsy/force/pull/5112/files#diff-d578803114a5017a93b60c336f9c9e33R14 but moved it to inside `await`.